package com.example.android_labs.Osrm

import com.example.android_labs.Osrm.OsrmResponse
import retrofit2.Response
import retrofit2.http.GET
import retrofit2.http.Path
import retrofit2.http.Query

interface OsrmApi {
    @GET("route/v1/foot/{start};{end}")
    suspend fun getRoute(
        @Path("start") start: String,
        @Path("end") end: String,
        @Query("overview") overview: String = "full"
    ): Response<OsrmResponse>
}